apps/bthome/settings.js

```diff
@@ -11,10 +11,24 @@
     require("Storage").writeJSON("bthome.json",settings)
   }
 
+  function getNewIdNumber(){
+    var id_start = 1;
+    var select = false;
+    while(!select){
+      select = true;
+      settings.buttons.forEach((button, idx) => {
+        if (id_start == button.n){
```

Comment:
Could the buttons ever go, `2, 1, 3`, for example?

Reply:
Yes it can, if you delete one of the buttons and it re-ranges it.